homeassistant/components/lock/__init__.py

```diff
@@ -31,9 +31,22 @@
 
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
 
+SERVICE_TO_METHOD = {
+    SERVICE_LOCK: 'lock',
+    SERVICE_UNLOCK: 'unlock'
+}
+
+ATTR_CODE = 'code'
+ATTR_CODE_FORMAT = 'code_format'
+
+ATTR_TO_PROPERTY = [
```

Comment:
These changes needs to be looked at together with
https://github.com/balloob/home-assistant-polymer/pull/25